Root/index.html

```diff
@@ -58,7 +58,7 @@
       </table>
     </div>
     <div>
-      
+      hello
```

Comment:
Remove hello.

Also in your CSS try and keep your spaces consistant. For example you have:
`background-color:orange;`
Try and always put a space after the colon like:
`background-color: orange;`

I know it's not strictly necessary, it's more for readability.